definitions/crds/crds.yaml

```diff
@@ -243,6 +243,8 @@ spec:
                     - kind
                     - name
                     properties:
+                      apiVersion:
+                        type: string
```

Comment:
Currently, we manually maintain the OpenAPI schema. In order to update the schema, we need to:
- update [chart](https://github.com/nirmata/kyverno/blob/master/charts/kyverno/crds/crds.yaml)
- update  `definitions/crds/crds.yaml` and run `make kustomize-crd`, this will update install.yaml and install_debug.yaml.

Reply:
Implemented the changes